eng/pipelines/doc-index.yml

```diff
@@ -0,0 +1,86 @@
+jobs:
+  - job: "Build"
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: "windows-2019"
+    steps:
+      - pwsh: |
+          Invoke-WebRequest -Uri "https://github.com/dotnet/docfx/releases/download/v2.43.2/docfx.zip" `
+          -OutFile "docfx.zip" | Wait-Process; Expand-Archive -Path "docfx.zip" -DestinationPath "./docfx/"
+        workingDirectory: $(Build.BinariesDirectory)
+        displayName: Download and Extract DocFX
+
+      - task: CmdLine@2
```

Comment:
I would use either pwsh or script to execute this command. No reason to use CmdLine.

Reply:
done